src/mui/list/Datagrid.js

```diff
@@ -53,7 +56,7 @@ class Datagrid extends Component {
                     {ids.map(id => (
                         <tr style={styles.tr} key={id}>
                             {React.Children.toArray(children).map((field, index) => (
-                                <TableRowColumn key={`${id}-${field.props.source || index}`}>
+                                <TableRowColumn key={`${id}-${field.props.source || index}`} style={{ padding: index ? '0 12px' : '0 12px 0 24px' }} >
```

Comment:
this should be a property of `styles` like the others [here](https://github.com/marmelab/admin-on-rest/blob/55ddcf01783d220a6896ec3b645bbc4f7a2d71ce/src/mui/list/Datagrid.js#L7)